### ISSUE TITLE
"New line between declarations" warning at top of each test file

### ISSUE THREAD

**Issue by abejfehr** [tlvince/eslint-plugin-jasmine#129]
**Do you want to request a *feature* or report a *bug*?**

I believe this is a bug, but I could be wrong about the expected behaviour.

**What is the current behavior?**

Currently, the first `describe()` call in any file is marked by eslint as if it's in violation of the `new-line-between-declarations` rule

**If the current behavior is a bug, please provide the steps to reproduce.**

Here's a screenshot of what I see in Atom, using the `linter-eslint` package:

<img width="762" alt="screen shot 2017-06-27 at 20 15 50" src="https://user-images.githubusercontent.com/3917428/27615378-b30b3958-5b75-11e7-9977-5980cff7e6ba.png">

And in the status bar:

<img width="905" alt="screen shot 2017-06-27 at 20 15 58" src="https://user-images.githubusercontent.com/3917428/27615383-bd99e41e-5b75-11e7-8e69-550fc57a3db2.png">

**What is the expected behavior?**

For the first line not to trigger this warning.

**Please mention your node.js, eslint-plugin-jasmine and operating system version.**

**Operating System:** macOS 10.12.5
**eslint:** 4.1.1
**eslint-plugin-jasmine:** 2.6.2
**linter-eslint** 8.2.1

**Comment by abejfehr** [comment]
Actually my mistake, I can't reproduce this with a simple test. Maybe there's something else wrong with the file and it's outputting this way. I'll keep this issue open for the next few hours until I figure out what's up.

Sorry for the noise if anyone's getting emails about this.

**Comment by abejfehr** [comment]
Closing. For those that are wondering, it was something like this:

``` 
beforeEach(module('some.module'));
beforeEach(inject((_$rootScope_, _$q_, _$componentController_, _$state_, $injector) => {
```

Since they didn't have a blank line between them there was a warning, but shown on line 1 for some reason.
